target/linux/rockchip/armv8/base-files/etc/board.d/02_network

```diff
@@ -66,6 +67,7 @@ rockchip_setup_macs()
 		wan_mac=$(get_mac_binary "/sys/bus/i2c/devices/2-0051/eeprom" 0xfa)
 		lan_mac=$(macaddr_setbit_la "$wan_mac")
 		;;
+	friendlyarm,nanopi-r3s|\
```

Comment:
please move it above `friendlyarm,nanopi-r4s-enterprise`

Reply:
What would be the reason for this? AFAIK the R3S does not have unique MAC addresses.

Reply:
Just reorder it alphabetically, like this:

```
	friendlyarm,nanopi-r3s|\
	friendlyarm,nanopi-r5c|\
	friendlyarm,nanopi-r6c|\
	friendlyarm,nanopi-r6s)
		wan_mac=$(macaddr_generate_from_mmc_cid mmcblk*)
		lan_mac=$(macaddr_add "$wan_mac" 1)
		;;
	friendlyarm,nanopi-r4s-enterprise)
		wan_mac=$(get_mac_binary "/sys/bus/i2c/devices/2-0051/eeprom" 0xfa)
		lan_mac=$(macaddr_setbit_la "$wan_mac")
		;;
```